moped-database/migrations/1703023666940_insert_into_public_moped_subphases/up.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO "public"."moped_subphases"("subphase_id", "subphase_name", "subphase_description", "related_phase_id") VALUES (28, E'Discovery', null, 1);
```

Comment:
You want to leave the `subphase_id` out of the insert and let the sequence take care of it, otherwise the sequence falls out of sync with the ID values and the sky will fall 💥

Reply:
okay i was wondering about that and had a suspicion this was the case, thanks for catching it!